content/docs/oracle/run-oracle/_index.md

```diff
@@ -34,63 +35,141 @@ Transaction costs for submitting price report data on-chain _are_ refunded but t
+   - `<PLUGINS_CONF_FILE>` is the path to the plugins YAML configuration file `plugins-conf.yml` (defaults to `./build/bin/plugins/plugins-conf.yml`).
 
 Oracle server will connect to external data sources using the providers in the `/plugins` subdirectory and begin submitting price reports to the connected node.
 
-{{< alert title="External data source plugins" >}}
+{{< alert title="Info" >}}
+See the [Autonity Oracle Server command-line reference](/reference/cli/oracle/#usage) for all available flags. Options can be set as system environment variables or directly in the terminal.
+
 By default oracle server will use data source providers packaged in the server release `/plugins` subdirectory.
 
-New plugins are configured by simply adding the binary code to the `/plugins` directory. See [Install Autonity Oracle Server](/oracle/install-oracle/) for how to do this.
+New plugins are configured by simply adding the binary code to the `/plugins` directory. See [Installing data source plugins](/oracle/install-oracle/#install-plugin).
 
-If plugins for external data sources or the symbols for which oracle server provides price data are changed while oracle server is running, the server does **not** need to be re-started: changes will be auto-detected and applied. 
+If plugins for external data sources or the symbols for which oracle server provides price data are changed while oracle server is running, changes are auto-detected and applied. Oracle server does **not** need to be re-started.
 {{< /alert >}}
 
-## Configure plugins
+## Run Autonity Oracle Server as Docker image {#run-docker}
+
+- Ensure that the Autonity Oracle Server [Docker image](/oracle/install-oracle/#install-docker) has been installed.
+
+1. Enter your working directory for autonity oracle server.
+
+2. Create and edit your oracle server config file `plugins-conf.yml` to specify the `name` and `key` for each plugins you are using.
+
+   ```bash
+   touch plugins-conf.yml
+   ```
+
+   {{< alert title="Info" >}}
+   A [sample `plugins-conf.yml` config file <i class='fas fa-external-link-alt'></i>](https://github.com/autonity/autonity-oracle/blob/master/config/plugins-conf.yml) can be downloaded from the Autonity Oracle Server GitHub.
+   {{< /alert >}}
+
+   Edit `plugins-conf.yml` to [configure plugins](/oracle/run-oracle/#configure-plugins) for data sources. See [Set up plugins config file](/oracle/run-oracle/#set-up-plugins-config-file) for how to do this.
+   
+3. Set the Docker configuration and the arguments for running Autonity Oracle Server and connecting to the Autonity Go Client it is serving.
+
+   ```bash
+   docker run \
+        -t -i \
+        --volume $<ORACLE_KEYFILE>:/autoracle/oracle.key \
+        --volume $<PATH_TO_PLUGINS_FILE>:/autoracle/plugins-conf.yml \
+        --name oracle-server \
+        --rm \
+        ghcr.io/autonity/autonity-oracle:latest \
+        -oracle_key_file="/autoracle/oracle.key" \
```

Comment:
Update flag naming again.

Reply:
Done, resolved in commit 7b5a773d9d7c2590fad714687b3a1fcdea79e704